### PR TITLE
docs: add type definition for ReminderManagementServiceClient

### PR DESCRIPTION
**Pull request by tianrenz**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have read the **README** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

## License

- [x] I confirm that this pull request can be released under the Apache 2 license

[issues]: https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/issues
[license]: http://aws.amazon.com/apache2.0/
[cla]: http://en.wikipedia.org/wiki/Contributor_License_Agreement

### CODE REVIEW

File: docs/en/Calling-Alexa-Service-APIs.rst [hunk 1]
```diff
@@ -581,4 +581,23 @@ Type Definition
...
     getSystemTemperatureUnit(deviceId: string): Promise<services.ups.TemperatureUnit>;
     getSystemTimeZone(deviceId: string): Promise<string>;
-  }
\ No newline at end of file
+  }
+
+ReminderManagementServiceClient
+===============================
+
+``ReminderManagementServiceClient`` can be used to query `Alexa Reminders API <https://developer.amazon.com/docs/smapi/alexa-reminders-api-reference.html>`_ to create and manager reminders from your skill.
```
Comment (breedloj) [inline docs/en/Calling-Alexa-Service-APIs.rst]: *create and manage

Comment (nikhilym) [inline docs/en/Calling-Alexa-Service-APIs.rst]: nit : type `manage`

Comment (tianrenz) [reply docs/en/Calling-Alexa-Service-APIs.rst]: fixed

Comment (tianrenz) [reply docs/en/Calling-Alexa-Service-APIs.rst]: fixed

**Review by nikhilym — APPROVED** [review]
🚢

**Review by breedloj — APPROVED** [review]

**Pull request merged**